### ISSUE TITLE
pathless route dosen't work with Solid router 

### ISSUE THREAD

**Issue by composite** [oedotme/generouted#83]
Pathless pattern feature in generouted dosen't work with solid router.
I've forked and created pathless route in example:

```
- src
  - pages
    - (app)
      - wow.tsx
    - _app.tsx
    - ...etc
```

Expected behavior: `/wow` route shows the page content like `Bow Wow`. (react router works fine)
Actual behavior: `/wow` route shows `404`.

`/wow` exists in `src/pages/routes.ts`.

Reproduce:
[Solid example source](https://github.com/composite/generouted/tree/solid/pathless/examples/solid-router)
[Stackblitz](https://stackblitz.com/github.com/composite/generouted/tree/solid/pathless/examples/solid-router)

**Comment by composite** [comment]
Um... I found why it does not work.

`path` is mandatory in Solid router. pathless route will not work if `path` is not present.

I found a workaround by editing `main.tsx` like:

```tsx
import { render } from 'solid-js/web'
import { Routes, routes } from '@generouted/solid-router'

// add path if pathless group exists.
routes.forEach((route: any) => {
  if (route?.id?.[0] == '(' && !route.path) route.path = '/'
})

render(Routes, document.getElementById('root'))

```

**Comment by oedotme** [comment]
Hey @composite, thanks for reporting the issue and the suggestion.

I thought it was working fine with solid, but apparently I was mistaken. I checked the PR you've submitted - thanks for that! I've left you a comment there. It should be fixed in [v1.12.5](https://github.com/oedotme/generouted/releases/tag/v1.12.5).
